README.md

```diff
@@ -392,6 +392,7 @@ Grist can be configured in many ways. Here are the main environment variables it
 | GRIST_ENABLE_SCIM                  | optional. If set, enable the [SCIM API Endpoint](https://support.getgrist.com/install/scim/) (experimental)                                                                                                                                                                                                                                                   |
 | GRIST_OIDC_...                  | optional. Environment variables used to configure OpenID authentification. See [OpenID Connect](https://support.getgrist.com/install/oidc/) documentation for full related list of environment variables.                                                                                                                                                                                                                                                  |
 | GRIST_SAML_...                  | optional. Environment variables used to configure SAML authentification. See [SAML](https://support.getgrist.com/install/saml/) documentation for full related list of environment variables.                                                                                                                                                                                                                                                  |
+| GRIST_IDP_EXTRA_PROPS                  | optional. If set, defines which extra fields returned by your identity provider will be stored in the users table of the home database (in the `options.sso_extra` object). Usage: 'onekey,anotherkey'.                                                                                                                                                                                                                                                 |
```

Comment:
If my remark below about the naming of the property has been taken into account, don't forget to adapt `options.sso_extra` in this piece of documentation :)